src/test/java/core/basesyntax/impl/FruitServiceImplTest.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.impl;
+
+import static core.basesyntax.model.FruitTransaction.Operation.BALANCE;
+import static core.basesyntax.model.FruitTransaction.Operation.PURCHASE;
+import static core.basesyntax.model.FruitTransaction.Operation.RETURN;
+import static core.basesyntax.model.FruitTransaction.Operation.SUPPLY;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationStrategy;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class FruitServiceImplTest {
+    private static FruitServiceImpl fruitService;
+
+    @Test
+    void fruitService_emptyData_Ok() {
+        fruitService = new FruitServiceImpl(new OperationStrategy(null));
```

Comment:
You have to move it in @BeforeAll
```suggestion
        fruitService = new FruitServiceImpl(new OperationStrategy(null));
```